### ISSUE TITLE
Mixed tabs/spaces does not sort properly

### ISSUE THREAD

**Issue by webdevnerdstuff** [Tyriar/vscode-sort-lines#74]
VS Code: v1.43.2
vscode-sort-lines: v1.9.0
OS: OsX Catalina

**Issue:**
Let me start with yes, I know we should not be using mixed tabs and spaces.

If lines have a mix of spaces and tabs it will sort the tabbed lines and spaced lines separate. I just ran into this issue because VSCode wasn't using the correct formatting for some reason and it was confusing me as to why they were not sorting correctly.

**Steps to replicate:**
1. Highlight rows
2. Push F9 or F1 and select sort option
3. Notice Items are sorted, but by indentation type separately.

**Ex.**
```css
.foo {
→  background-color: #fff;
→  border-radius: 6px;
→  height: auto;
→  width: 340px;
•• margin-top: 15px;
•• opacity: 0;
}
```

**Comment by Tyriar** [comment]
The extension doesn't know anything about indentation right now, seems like the extension is doing a great service by pointing out the inconsistency. You can use editorconfig or set up a formatter so that fixing indentation file-wide is as simple as option+shift+f

**Comment by webdevnerdstuff** [comment]
I know the extension doesn't know about indentation, that is the exact reason for reporting this behavior. If you reread my original comments I explained how and why I noticed the behavior. I understand you think it's a good thing, but I tried to explain to you why it could be problematic.

Perhaps as a way to improve upon things, it could add a space between the spaces and indented lines? This might help to avoid confusion, as a user could think the plugin is not working correctly. Or at the very least add to the documentation that this is an intended feature? 

While I appreciate your suggestions, I am aware of editorconfig and formatting plugins, as I use them with all projects I work on. They were both being used when I noticed the behavior. Something was wrong with VSCode when this problem happened, and went back to its expected functionality after I realized the issue and restarted the program.
